app/controllers/users.js

```diff
@@ -3,18 +3,22 @@ const errors = require('../errors');
 const { createHash, validateWithHash } = require('../helpers/hashing');
 const { CREATED, SIGN_IN_SUCCESSFUL, BAD_CREDENTIALS, GET_USERS_OK } = require('../../config/constants');
 const { getToken } = require('../services/sessions');
+const MailerService = require('../services/mailer');
+const logger = require('../logger');
 
 const createUser = async (req, res, next) => {
   try {
     const { body: userData } = req;
     const hashCode = await createHash(userData.password);
     if (hashCode) userData.password = hashCode;
     await UserServices.createUser(userData);
+    await MailerService.sendWelcomeEmail(userData);
     return res.status(201).send({
       data: { name: userData.name },
       message: CREATED
     });
   } catch (err) {
+    logger.info('createUser: ', err);
```

Comment:
You're logging the error or the created user?

Reply:
Fixed